cmd/ocitools/validate.go

```diff
@@ -61,6 +62,8 @@ var (
 		"CAP_KILL",
 		"CAP_AUDIT_WRITE",
 	}
+
+	configSchemaTemplate = "https://raw.githubusercontent.com/opencontainers/runtime-spec/v%s/schema/schema.json"
```

Comment:
On Thu, Sep 01, 2016 at 03:37:26PM -0700, Mrunal Patel wrote:
> > @@ -61,6 +62,8 @@ var (
> >  		"CAP_KILL",
> >  		"CAP_AUDIT_WRITE",
> >  	}
> > +
> > +	configSchemaTemplate = "https://raw.githubusercontent.com/opencontainers/runtime-spec/v%s/schema/schema.json"
> 
> If we are compiling ocitools for a particular runtime-spec version…

I'd rather *not* compile ocitools for a particular runtime-spec
version.  By staying flexible here we have one less place where we are
pinning ourselves down.